### ISSUE TITLE
More info on how to use

### ISSUE THREAD

**Issue by KennyDenvers** [Rudtrack/structured-tree#1]
This plugin looks interesting and useful! I am not really sure how to use it in practice, though. Can you point me to any resources that could give me more info in this direction? Also: Can I integrate existing Obsidian notes into the plugin "dendron" hierarchy?

**Comment by Rudtrack** [comment]
Thank you! I was thinking of writing up some documentation very soon, with some examples of how I personally use this plugin. I'll update this issue when there is something to show!

**Comment by KennyDenvers** [comment]
Thanks, that helps for a start. I am looking forward to your documentation. 
Maybe you have one more hint for me: I have tried to rename some notes, trying to connect them with the dot ".". I tried to define one as the "mother note" (or folder note) and tried to rename it like this "mother_note.child_note", but this did not seem to do anything to either of them.

**Comment by Rudtrack** [comment]
Are you viewing it through the Structured Tree explorer, or the regular file explorer for Obsidian?

![bilde](https://github.com/user-attachments/assets/7e220297-8ca4-4215-893f-7a6f4a5939ce)

---

Have you added the folder as a vault? The plugin doesn't go through subfolders, so every folder needs to be added here. Ideally you should have all your notes in one folder for this plugin to work as intended

![bilde](https://github.com/user-attachments/assets/f5c4e92b-ead0-4b50-ad76-ce1c86a733ea)

Example: You have a folder structure `Top Folder` --> `Child Folder`. If you only add `Top Folder` as a vault, only notes located inside `Top Folder` will be detected. No notes in `Child Folder` will show up unless you add that folder aswell

---

I highly recommend taking a backup of all your notes in your current organizational structure before trying to switch to this plugin. Would be a real bad time to convert everything, then find out you don't like it and having to undo it all manually!

**Comment by KennyDenvers** [comment]
Thanks for your support and advice!
I do have folders. In the "Structured Tree" view I see all the notes in my root folder. For testing purposes I tried to "connect" notes that are located in the root folder by trying to tie them together by renaming the "mother_note.child_note". Should that work on existing notes quite generally? I would like to try out the "structured tree" method in some topical areas of my vault where i can see a lot of benefit from this kind of structure.

**Comment by Rudtrack** [comment]
Is `mother_note` a folder or a file? For it to work, it should be a file, as every level in the hiearchy should be a file (or nothing). I've also updated the [Wiki](https://github.com/Rudtrack/structured-tree/wiki) with more information, so maybe something there can help you as well!

---

Linking (with the Obsidian backlinks `[[]]`) seems to strugle a bit right now. At least when you have the `Custom Resolver` option enabled, it will create a new page in your current vault instead of taking you to a new vault. This plugin was originally designed to work with [Dendron](https://www.dendron.so/), and that was the behavior for that program. I forked the original plugin, because I wanted to further develop this style of note-taking in Obsidian. 

But as a result, using multiple vaults can be a bit clunky if you're trying to get them to interact with each other. This is of course something I will look at in the future!
